docs/features/diff-viewer/diff-viewer.md

```diff
@@ -11,10 +11,11 @@ The following sections describe how you can create, browse, configure, manage an
 1. [Create Comparison Manually](#create-comparison-manually)
 2. [Select Comparison to Browse](#select-comparison-to-browse)
 3. [Browse Comparison Differences](#browse-comparison-differences)
-4. [Hide Comparison](#hide-comparisons)
-3. [Configuration of Automatic Comparisons](#configuration-of-automatic-comparisons)
-4. [Manage Comparisons](#manage-comparisons)
-5. [Delete Comparisons](#delete-comparisons)
+4. [Hide Comparisons](#hide-comparisons)
+5. [Configuration of Automatic Comparisons](#configuration-of-automatic-comparisons)
+6. [Manage Comparisons](#manage-comparisons)
+7. [Integrate Comparisons in CI Pipeline](#integrate-comparisons-in-ci-pipeline)
```

Comment:
Please put this point below point 5 - since it is another way to trigger comparsions (similar to 5) and the manage view is less important.